src/database/data/networks.ts

```diff
@@ -835,7 +835,8 @@ export const NETWORKS_CONFIG: NetworkConfig = {
     tokens: [],
     contracts: [],
     dPaths: {
-      [WalletId.MNEMONIC_PHRASE]: DPaths.AUX_DEFAULT
+      [WalletId.MNEMONIC_PHRASE]: DPaths.AUX_DEFAULT,
+      [WalletId.TREZOR]: DPaths.AUX_DEFAULT
```

Comment:
Please also add your DPath to the `TREZOR_NEW` and `MNEMONIC_PHRASE_NEW` WalletId in order to be supported on our latest hardware wallet flow.